CPackProperties.cmake.in

```diff
@@ -0,0 +1,12 @@
+# Make the current version available to CPack
+set(CPACK_PACKAGE_VERSION "@REV_VERSION@")
```

Comment:
The `REV_VERSION` is based on the git tag(project version) or the git commit and can change between builds.
This is not pretty but necessary to make CPack behave correctly.

Reply:
Please not that this will usually not change between builds: CMake will not notice if your git status has changed, so it will not rerun CMake, so it will not rerun your ```execute_process()```.